policy/policy-authority.go

```diff
@@ -62,6 +63,28 @@ func suffixMatch(labels []string, suffixSet map[string]bool, properSuffix bool)
 	return false
 }
 
+// Given a domain name, return a truncated name of the form <label>.PSL
```

Comment:
Style: Please start the comment with "PSLPlusOne returns a truncated name.... given a domain name" per the Go lint complain-o-tron.